Data/models/DataController.swift

```diff
@@ -96,5 +89,19 @@ public class DataController: NSObject {
         
         container.persistentStoreDescriptions = [storeDescription]
     }
+    
+    let storeURL: URL = {
+        let urls = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)
+        guard let docURL = urls.last else {
+            log.error("Could not load url at document directory")
+            fatalError()
+        }
+        let storeURL = docURL.appendingPathComponent(DataController.databaseName)
```

Comment:
Small nit: You could just return the `docURL.appendingPathComponent(DataController.databaseName)` since you don't use `storeURL`

Reply:
👍 I sometimes do this to document what the whole `docURL.append...` stuff _is_. However, as you point out, totally unnecessary here, as that is what the prop itself is named. Fixed.